frontend/src/components/NavBar.tsx

```diff
@@ -33,6 +41,17 @@ export const NavBar = () => {
         </div>
         <RoadmapSelectorWidget />
         <div className={classes(css.navBarDivider)} />
+        {userInfo && (
+          <div className={classes(css.userInfoContainer)}>
+            <div className={classes(css.memberIcon)}>
+              {userRole === RoleType.Admin && <StarSharpIcon />}
+              {userRole === RoleType.Developer && <BuildSharpIcon />}
+              {userRole === RoleType.Business && <BusinessIcon />}
```

Comment:
```suggestion
              {userRole === RoleType.Business && (
                <BusinessIcon color={colors.black100} size="small" />
              )}
```

Businessikoni on tällä hetkellä harmaa. Myös sen kokoa voisi pienentää